src/include/execution/sql/functions/comparison_functions.h

```diff
@@ -216,6 +216,14 @@ class EXPORT ComparisonFunctions {
     return int32_t(len1) - int32_t(len2);
   }
 
+  /**
+   * Logical negation for SQL booleans
+   */
+  static void NotBoolVal(terrier::execution::sql::BoolVal *result, terrier::execution::sql::BoolVal *input) {
```

Comment:
1. Remove the namespaces.
2. Change second argument (i.e., input) to `const &` to convey intention.
3. Simplify logic:
```
result->is_null_ = input.is_null_;
result->val_ = !input.val_
```